test/parallel/test-net-write-arguments.js

```diff
@@ -31,4 +39,10 @@ assert.throws(() => socket.write(null),
     message: 'The "chunk" argument must be of type string or an instance of ' +
               `Buffer.${common.invalidArgTypeHelper(value)}`
   }));
+  socket.on('error', common.expectsError({
+    code: 'ERR_INVALID_ARG_TYPE',
+    name: 'TypeError',
+    message: 'The "chunk" argument must be one of type string or Buffer. ' +
+              `Received type ${typeof value}`
```

Comment:
The error message recently changed and is now: `'The "chunk" argument must be of type string or an instance of Buffer. Received type boolean (true)'`

See https://ci.nodejs.org/job/node-test-commit-linuxone/18077/nodes=rhel7-s390x/testReport/junit/(root)/test/parallel_test_net_write_arguments/